src/main/java/seedu/address/model/person/passenger/Passenger.java

```diff
@@ -75,10 +75,18 @@ public TripDay getTripDay() {
         return tripDay;
     }
 
+    public String getTripDayStr() {
```

Comment:
small thing but, perhaps this could be renamed getTripDayAsString() as we have discussed